sigstore/_cli.py

```diff
@@ -151,6 +143,22 @@ def main() -> None:
     show_default=True,
     help="The Rekor instance to use (conflicts with --staging)",
 )
+@click.option(
+    "ctfe_pem",
+    "--ctfe",
+    metavar="FILE",
+    type=click.File("rb"),
+    default=BytesIO(DEFAULT_REKOR_CTFE_PUBKEY),
+    help="A PEM-encoded public key for the CT log (conflicts with --staging)",
+)
+@click.option(
+    "rekor_root_pubkey",
+    "--rekor-root-pubkey",
+    metavar="FILE",
+    type=click.File("rb"),
+    default=BytesIO(DEFAULT_REKOR_ROOT_PUBKEY),
+    help="A PEM-encoded root public key for Rekor itself (conflicts with --staging)",
+)
```

Comment:
Flagging for review: this is like https://github.com/sigstore/sigstore-python/pull/121/files#r891786806 below, except I went ahead and added the needed flag since it's less complicated. But I can remove it and have it return an error for the time being, instead, if desired.